ansible_collections/arista/avd/roles/eos_designs/docs/how-to/avd-to-cvaas.md

```diff
@@ -13,8 +12,12 @@ Specify the "valid until" date.
 Make sure to copy the generated password. You only get view it once.
 Click "Save" to exit the dialogue box. 
 ```
-![Figure: 2](../../../../media/account_settings.png)
 
+![Figure: 1](../../../../media/serviceaccount1.png)
+![Figure: 2](../../../../media/serviceaccount2.png)
+![Figure: 3](../../../../media/serviceaccount3.png)
+
+> NOTE The name of the service account must match a username configured to be authorized on EOS, otherwise device interactive API calls might fail due to authorization denial.
 ## Add CVAAS to your Ansible inventory file:
```

Comment:
add new line before the heading

Reply:
I forget but I think we don't need punctuation marks in titles in general, @JulioPDX can you confirm please?

Reply:
agreed